CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
```

Comment:
I amended. Since I couldn't specify the date of the new version, I moved that under `UNRELEASED`. I think that the process should be now following:
- after you merge this, I will push new commit with updated changelog where unreleased stuff will be moved after proper version with date
- I will push tag/release a new version
- I will rebase the other MR and fix changelog
- repeat steps 1 and 2 for other version 2.0.1

Do you agree or is there any better workflow how to do this? I don't really like pushing the updated changelog in separate commit to master, but I am not sure what is the best practice to do that, since there is no guarantee that it will be actually merged and released the same day as stated in the changelog :/